src/rust/engine/fs/src/directory.rs

```diff
@@ -893,6 +1019,7 @@ impl From<&DigestTrie> for remexec::Tree {
     trie.walk(&mut |_, entry| {
       match entry {
         Entry::File(_) => (),
+        Entry::Symlink(_) => todo!(),
```

Comment:
Do nothing. This loop gathers Directory protos so they can be consolidated into a `Tree` proto. (Metadata for files and symlinks will be stored in those `Directory` protos.)